From 3a8de744d45dda62ce70eb153eb0c7348bfae750 Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Fri, 21 Mar 2025 10:10:42 -0700
Subject: [PATCH 6/8] printout aotriton tuning db gpu info

Printout information if tuning data is available
in database for kernel when build for the selected gpu target.

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 v2python/tuning_database.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/v2python/tuning_database.py b/v2python/tuning_database.py
index 4dedf17..7c433f2 100644
--- a/v2python/tuning_database.py
+++ b/v2python/tuning_database.py
@@ -3,6 +3,8 @@
 
 import pathlib
 import sqlite3
+import sys
+import time
 from .kernel_argument import ArgumentSelection, TunedArgument
 from .gpu_targets import AOTRITON_GPU_ARCH_TUNING_STRING, AOTRITON_TUNING_DATABASE_REUSE
 from .common_tuning_database import CommonKernelTuningDatabaseForArch
@@ -122,6 +124,8 @@ class KernelTuningDatabase(object):
                 self.arch_dict[arch] = EmptyKernelTuningDatabaseForArch(self._kdesc, arch)
             else:
                 self.arch_dict[arch] = BootstrapTuningDatabaseForArch(self._kdesc, arch)
+        else:
+            print(f'For kernel {self._kdesc.KERNEL_FAMILY}.{self._kdesc.name}, Architecture {arch} was found in tuning database')
         return self.arch_dict[arch].set_gpu(gpu, index)
 
     @property
-- 
2.48.1

